makefile

```
@@ -276,6 +279,9 @@ glvis:	glvis.cpp lib/libglvis.a $(CONFIG_MK) $(MFEM_LIB_FILE)
 	$(CCC) -o glvis glvis.cpp -Llib -lglvis $(LIBS)
 
 $(LOGO_FILE_CPP): $(LOGO_FILE)
+ifndef XXD_FOUND
+	$(error Required tool not found: xxd. Stop. $(XXD_FOUND))
```

Make automatically adds `.  Stop.` to `$(error ...)` commands, so I suggest we just use this:
```suggestion
	$(error Required tool not found: xxd)
```
which prints:
```console
makefile:283: *** Required tool not found: xxd.  Stop.
```